docs/sphinx/install.rst

```diff
@@ -141,6 +141,41 @@ we refer to the `CUDA Quantum GitHub repository`_.
 
 .. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
 
+
+CUDA Quantum Dependencies
+-------------------------
+
+CUDA Quantum can be used to simulate quantum programs (see `using/simulators`) on a CPU-only system, but a GPU is highly recommended.
+
+The supported CPUs include both the x86_64 (x86-64-v3 when using the OpenMPI stack) and ARM64 architectures.
+
+For simulation of CUDA Quantum programs on NVIDIA GPUs, additional packages should be installed. See also `cuQuantum <https://docs.nvidia.com/cuda/cuquantum/latest/getting_started.html>`__.
+
+The following table summarizes the required components.
+
+.. list-table:: Supported Systems
+    :widths: 30 50
+    :header-rows: 0
+
+    * - CPU architectures
+      - x86_64, ARM64
+    * - Operating System
+      - Linux
+    * - Tested Distributions
+      - Ubuntu 22.04
+
+.. list-table:: Requirements for GPU simulation
```

Comment:
nit: since `Systems` is capitalized in line 156, I think either `Simulation` should be capitalized here, or `Systems` should not be capitalized in line 156.
```suggestion
.. list-table:: Requirements for GPU Simulation
```